lib/valueflow.cpp

```diff
@@ -5998,7 +5987,7 @@ static void insertNegateKnown(std::list<ValueFlow::Value>& values, const std::li
 
 struct ConditionHandler {
     struct Condition {
-        const Token *vartok;
+        const Token* vartok{};
         std::list<ValueFlow::Value> true_values;
         std::list<ValueFlow::Value> false_values;
         bool inverted = false;
```

Comment:
Why explicit `false` here?

Reply:
AFAIK we have not yet decided on a mandatory convention for initializers.